packages/cli/src/index.ts

```diff
@@ -14,6 +25,27 @@ program
     initCommand();
   });
 
+program
+  .command('sync')
+  .option(
+    '-n, --novu-cloud-api-url <novuCloudApiUrl>',
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** ‏ renamed parameter, let me know what you think